test/force-auto.py.in

```diff
@@ -130,6 +130,10 @@ class Test(abc.ABC):
       # architecture excludes
       arch_excludes = "\n".join("arch_exclude %s" % i
                                 for i in self.arch_excludes)
+      if ((str(self.base) == "alpine:latest") and (str(self.run.value) == "needed") and (str(self.force) == "fakeroot") and (not preprep)):
```

Comment:
Don't stringify unless needed. E.g. something like:

```suggestion
      if (    self.base == "alpine:latest"
          and self.run == Run.NEEDED
          and self.force == "fakeroot"
          and not preprep):
```

That said, this is a very specific exception with no explanation. I think you are dealing with a symptom, not a cause. I recommend you investigate this test and see if it can be updated to not require `--rebuild`.